### PR TITLE
Upgrade to Django 1.11

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Fix HURUmap to use Django 1.11

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -6,5 +6,5 @@ PGDATABASE=hurumap
 PGUSER=hurumap
 PGPASSWORD=hurumap
 PYTHONDONTWRITEBYTECODE="True"
-USE_MAPIT="True"
+USE_MAPIT=True
```
Comment (kilemensi) [inline .env.example]: Change `PYTHONDONTWRITEBYTECODE="True"` to `PYTHONDONTWRITEBYTECODE=True` as well.

File: Dockerfile [hunk 2]
```diff
@@ -38,7 +38,7 @@ WORKDIR $APP_SRVPROJ
 
 # Workaround for dependency links
 # Comment out when changes / updates / PR accepted
-RUN pip install git+https://github.com/CodeForAfricaLabs/wazimap.git@openup/releases#egg=wazimap[gdal]
+RUN pip install git+https://github.com/OpenUpSA/wazimap.git@releases#egg=wazimap[gdal]
```
Comment (kilemensi) [inline Dockerfile]: The best approach is to update `CodeForAfricaLabs/wazimap/openup/releases` to `OpenUpSA/wazimap/releases` and leave this line unchanged.

File: hurumap/urls.py [hunk 3]
```diff
@@ -5,4 +5,4 @@
 from hurumap import settings
 
 urlpatterns = static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + \
-              wazimap_urlpatterns
+              list(wazimap_urlpatterns)
```
Comment (kilemensi) [inline hurumap/urls.py]: `static` returns a list of urls and `wazimap_urlpatterns` is also a list of urls (if upgraded to Django 1.11 properly) so this should work without an explicit cast to list. If it doesn't work, then you've uncovered a bug in `OpenUpSA/wazimap/releases`

Comment (phillipahereza) [reply hurumap/urls.py]: @kilemensi `urlpatterns` in wazimap are a tuple. That was the reason for the explicit cast to list

Comment (kilemensi) [reply hurumap/urls.py]: @phillipahereza 

From Django 1.8, the use of `turple`/`pattern()` was removed and `urlpatterns` are not expected to be a list: https://docs.djangoproject.com/en/2.1/releases/1.8/#django-conf-urls-patterns

Checkout `wazimap/master`, it's been upgraded to Django 1.11 as well and there, the `urlpatterns` were converted to a list... So if this explicit cast works, lets go ahead with it **but** this means something was left out in the `OpenUpSA/wazimap/releases` upgrade.

Comment (phillipahereza) [reply hurumap/urls.py]: @kilemensi I've updated our releases branch and opened a PR with the upstream to fix above issue

**Comment by phillipahereza** [comment]
@kilemensi can I merge this?

File: hurumap/settings.py [hunk 4]
```diff
@@ -252,7 +252,7 @@
 # -------------------------------------------------------------------------------------
 
 DATABASE_URL = os.environ.get('DATABASE_URL',
-                              'postgresql://hurumap:hurumap@localhost/hurumap')
+                              'postgresql://hurumap:hurumap@localhost/hurumap_ke')
```
Comment (kilemensi) [inline hurumap/settings.py]: Must be `hurumap`

File: hurumap/urls.py (hunk 3 above)
Comment (kilemensi) [inline hurumap/urls.py]: If you haven't, add a story in PT to track this.

Comment (phillipahereza) [reply hurumap/urls.py]: @kilemensi I added a story for that

**Pull request merged**